CMakeLists.txt

```diff
@@ -84,6 +92,12 @@ option(WITH_JAEGER "Whether to include the Jaeger exporter" OFF)
 
 option(BUILD_TESTING "Whether to enable tests" ON)
 if(WIN32)
+  if(BUILD_TESTING)
+    if(MSVC)
+      # GTest bug: https://github.com/google/googletest/issues/860
+      add_compile_options(/wd4275)
```

Comment:
Could we scope this option to only test code compilation?

Reply:
Unfortunately the issue is that we have so many different test projects, and it appears like it would have to be applied to all project that includes Google Test. The fix is in the latest Google Test. But that latest Google Test is not getting pulled via `vcpkg`, so I see this error. It'd have to be added to many files. I can do that if you insist